src/main/java/adris/altoclef/util/ProjectileUtil.java

```diff
@@ -1,15 +1,17 @@
 package adris.altoclef.util;
 
+
 import net.minecraft.client.network.ClientPlayerEntity;
 import net.minecraft.entity.projectile.ExplosiveProjectileEntity;
 import net.minecraft.entity.projectile.ProjectileEntity;
-import net.minecraft.util.math.MathHelper;
-import net.minecraft.util.math.Vec2f;
 import net.minecraft.util.math.Vec3d;
 
-public class ProjectileUtil {
 
-    public static final double GRAVITY_ACCEL = 0.05000000074505806D;
+public final class ProjectileUtil {
+    public static final double GRAVITY_ACCEL = 0.05000000074505806D; // TODO: 2021-05-22 magic constant?
```

Comment:
Yeah I think I copied this from minecraft's decompiled source code, I don't think this is publicly accessible (but if it is, would definitely switch to that)

Reply:
how is it derived? Also, it's best to move magic constants to a "MagicConstants" class imo.

Reply:
No clue, would have to do some research (I found this value when searching for a gravity acceleration value for arrows I think?)

I'll create "MagicConstants" or something like that when I find more examples. One example isn't enough to make a whole class, but I'll keep this in mind.